cli/pkg/kctrl/cmd/core/help_sections.go

```diff
@@ -17,6 +17,36 @@ var (
 		Value: "package",
 		Title: "Package Commands:",
 	}
+	PackageRepoHelpGroup = cobrautil.HelpSection{
+		Key:   cmdGroupKey,
```

Comment:
cmdGroupKey is "kapp-group" on line number 11, should we change it now to "kctrl-group"?

Reply:
Yep ! We should